tests/gnrc_udp/udp.c

```diff
@@ -208,19 +221,27 @@ int udp_cmd(int argc, char **argv)
     }
 
     if (strcmp(argv[1], "send") == 0) {
+        char *conversion_end;
         uint32_t num = 1;
         uint32_t delay = 1000000LU;
+        if (argc > 5) {
+            num = strtol(argv[5], &conversion_end, 0);
+            if (*conversion_end != '\0') {
+                goto send_syntax_err;
```

Comment:
Rather use a `_usage()` function here and just `return 1`. Jumping into the middle of some `if`-handling just invites spaghetti code.